bundles/automation/org.eclipse.smarthome.automation.core/OSGI-INF/RuleRegistry.xml

```diff
@@ -8,21 +8,31 @@
     http://www.eclipse.org/legal/epl-v10.html
 
 -->
-<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" activate="activate" modified="modified" deactivate="deactivate" immediate="true" name="org.eclipse.smarthome.automation.ruleregistry">
-   
-   <implementation class="org.eclipse.smarthome.automation.core.internal.RuleRegistryImpl"/>
-   
-   <service>
-      <provide interface="org.eclipse.smarthome.automation.RuleRegistry"/>
-   </service>
-   
-   <property name="rule.reinitialization.delay" type="Long" value="500"/>
-   
-   <reference bind="setTemplateRegistry" cardinality="1..1" interface="org.eclipse.smarthome.automation.template.TemplateRegistry" name="TemplateRegistry" policy="static" unbind="unsetTemplateRegistry"/>
-   <reference bind="setModuleTypeRegistry" cardinality="1..1" interface="org.eclipse.smarthome.automation.type.ModuleTypeRegistry" name="ModuleTypeRegistry" policy="static" unbind="unsetModuleTypeRegistry"/>
-   
-   <reference bind="setStorageService" cardinality="0..1" interface="org.eclipse.smarthome.core.storage.StorageService" name="StorageService" policy="dynamic" unbind="unsetStorageService"/>
-   <reference bind="setManagedProvider" cardinality="0..1" interface="org.eclipse.smarthome.automation.ManagedRuleProvider" name="ManagedRuleProvider" policy="dynamic" unbind="removeManagedProvider" />
-   <reference bind="setEventPublisher" cardinality="0..1" interface="org.eclipse.smarthome.core.events.EventPublisher" name="EventPublisher" policy="dynamic" unbind="unsetEventPublisher"/>
-   <reference bind="addModuleHandlerFactory" cardinality="0..n" interface="org.eclipse.smarthome.automation.handler.ModuleHandlerFactory" name="ModuleHandlerFactory" policy="dynamic" unbind="removeModuleHandlerFactory"/>
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" activate="activate" modified="modified"
```

Comment:
While you are changing the other XML file to be auto-generated via DS-annotation, please change this one too, thank you!